tests/test_flows_seed.py

```diff
@@ -734,3 +734,151 @@ def expect_unsupported_derivation(load_message: Callable):
+
+        self.run_sequence(
+            initial_destination_view_args=dict(seed_num=0),
+            sequence=[
+                FlowStep(seed_views.SeedOptionsView, button_data_selection=seed_views.SeedOptionsView.BIP352_SILENT_PAYMENTS),
+                FlowStep(seed_views.SeedBIP352SilentPaymentsOptionsView, button_data_selection=seed_views.SeedBIP352SilentPaymentsOptionsView.GENERATE_SP_ADDRESS),
+                FlowStep(seed_views.SeedBIP352LabelPromptView, button_data_selection=seed_views.SeedBIP352LabelPromptView.YES),
+                FlowStep(seed_views.SeedBIP352LabelEntryView, screen_return_value="123"),
+                FlowStep(seed_views.SeedBIP352GeneratePaymentAddressView, screen_return_value=RET_CODE__BACK_BUTTON),
+                FlowStep(seed_views.SeedBIP352LabelEntryView, screen_return_value="0"),
+                FlowStep(seed_views.SeedBIP352GeneratePaymentAddressView, button_data_selection=seed_views.SeedBIP352GeneratePaymentAddressView.EXPORT_VIA_QR_CODE),
+                FlowStep(seed_views.SeedBIP352PaymentAddressQRView, screen_return_value=0),
+                FlowStep(seed_views.SeedBIP352SilentPaymentsOptionsView),
+            ]
+        )
+
+
+    def test_bip352_export_spend_pubkey_flow(self):
+        mnemonic = "blush twice taste dawn feed second opinion lazy thumb play neglect impact".split()
+        self.controller.storage.set_pending_seed(Seed(mnemonic=mnemonic))
+        self.controller.storage.finalize_pending_seed()
+
+        self.settings.set_value(SettingsConstants.SETTING__BIP352_SILENT_PAYMENTS, SettingsConstants.OPTION__ENABLED)
+
+        self.run_sequence(
+            initial_destination_view_args=dict(seed_num=0),
+            sequence=[
+                FlowStep(seed_views.SeedOptionsView, button_data_selection=seed_views.SeedOptionsView.BIP352_SILENT_PAYMENTS),
+                FlowStep(seed_views.SeedBIP352SilentPaymentsOptionsView, button_data_selection=seed_views.SeedBIP352SilentPaymentsOptionsView.EXPORT_SPEND_PUBKEY),
+                FlowStep(seed_views.SeedBIP352ExportSpendPubkeyDetailsView, screen_return_value=0),
+                FlowStep(seed_views.SeedBIP352ExportSpendPubkeyQRView, screen_return_value=0),
+                FlowStep(seed_views.SeedBIP352SilentPaymentsOptionsView),
+            ]
+        )
+
+
+    def test_bip352_export_scan_privkey_flow(self):
+        mnemonic = "blush twice taste dawn feed second opinion lazy thumb play neglect impact".split()
+        self.controller.storage.set_pending_seed(Seed(mnemonic=mnemonic))
+        self.controller.storage.finalize_pending_seed()
+
+        self.settings.set_value(SettingsConstants.SETTING__BIP352_SILENT_PAYMENTS, SettingsConstants.OPTION__ENABLED)
+
+        self.run_sequence(
+            initial_destination_view_args=dict(seed_num=0),
+            sequence=[
+                FlowStep(seed_views.SeedOptionsView, button_data_selection=seed_views.SeedOptionsView.BIP352_SILENT_PAYMENTS),
```

Comment:
I probably would have just directly connected these tests into one sequence.

The strict separation here is slightly more rigorous, of course, just a tiny bit unnecessary.